config/initializers/secure_headers.rb

```diff
@@ -35,25 +36,24 @@
     ],
     style_src: ["'self'", IdentityConfig.store.asset_host.presence],
     base_uri: ["'self'"],
+    preserve_schemes: true,
+    disable_nonce_backwards_compatibility: true,
```

Comment:
I waiver on this. The change is here to match the default behavior for the rails tooling, but it will steer us into another "what browsers do we support" conversation

Reply:
> another "what browsers do we support" conversation

Based on [this](https://content-security-policy.com/nonce/#:~:text=Nonce%20Browser%20Support,browser%20for%20nonce%20support%20instead.), seems like it's a very limited slice of browsers which would be impacted? i.e. pre-2015 Chrome/Firefox, and Safari 9 and below (15 is current).